web/matrixdef

```diff
@@ -2764,12 +3010,14 @@ BeginIter adp{i} "an Adposition" 1
 
 EndIter adp
 
+
+
 Section morphology "Morphology"
 
 Cache nouns noun[0-9]+$ name
 Cache verbs verb[0-9]+$ name
+Cache qverbs qverb[0-9]+$ name
 Cache auxes aux[0-9]+$ name
-Cache dets det[0-9]+$ name
```

Comment:
Was this not being used?

Reply:
Hmm. This does not ring a bell. I remember removing something that used to be called "qdet" but not "det". (I wish we could have a few automated tests for the questionnaire... right now it is tricky because the website often isn't even working :) ).

Reply:
Thanks for this, T.J., this should of course stay.